mantle/cmd/kola/kola.go

```diff
@@ -588,19 +594,14 @@ func downloadImageAndDecompress(url, compressedDest string) (string, error) {
 	return compressedDest, nil
 }
 
-func getParentFcosBuildBase() (string, error) {
+// Returns the URL to a parent build that can be used as a base for upgrade
+// testing.
+func getParentFcosBuildBase(stream string) (string, error) {
 	// For FCOS, we can be clever and automagically fetch the metadata for the
 	// parent release, which should be the latest release on that stream.
 
-	// We're taking liberal shortcuts here... the cleaner way to do this is
-	// parse commitmeta.json for `fedora-coreos.stream`, then fetch the stream
-	// metadata for that stream, then fetch the release metadata
-
-	if kola.CosaBuild.Meta.BuildRef == "" {
-		return "", errors.New("no ref in build metadata")
-	}
-
-	stream := filepath.Base(kola.CosaBuild.Meta.BuildRef)
+	// We're taking liberal shortcuts here... the cleaner way to do this is to
```

Comment:
The code was moved but the comment got duplicated.

Reply:
The comments are slightly different from each other. So it's more of a split. :)